azure-pipelines.yml

```diff
@@ -159,3 +162,12 @@ stages:
               resourceGroup: $[dependencies.Terraform.outputs['terraformOutput.resourceGroup']]
               storageAccount: $[dependencies.Terraform.outputs['terraformOutput.storageAccount']]
               storageContainer: $[dependencies.createStorage.outputs['createContainer.container']]
+    
+    - ${{ if ne(environment.name, 'dev') }}:
+      - template: pipeline/jobs/sonarqubetest.yaml
+        parameters:
+          displayName: 'Sonarcloudtest'
+          dependsOn:
+          - Terraform
+          - createStorage
+          command: sonarqube --stacktrace -$(Sonarcloud_login)       
```

Comment:
use library variable, no need to reassign it

Reply:
used library variable and removed reassign variable